addons/sourcemod/scripting/shop/item_manager.sp

```diff
@@ -820,19 +821,12 @@ public int ItemManager_KvCopySubKeysItemCustomInfo(Handle plugin, int numParams)
 	KeyValues origin_kv = GetNativeCell(2);
 	if(origin_kv == INVALID_HANDLE) 
 		ThrowNativeError(SP_ERROR_NATIVE, "Handle is invalid!");
-	
-	bool result = false;
-	
-	if (h_KvItems.JumpToKey("CustomInfo", true))
```

Comment:
`KeyValues.JumpToKey(key, true)` - always returns true, because the key is created and it is, in fact, there.

Reply:
Theoretically, not always. False if the key could not be created

Reply:
If in theory, then various characters that do not refer to letters / numbers can possibly hurt, especially you should be careful with `/`, because this is a separator for keys.
But in our case, we are jumping to "CustomInfo", there should be no problems at all.